google-cloud-clients/google-cloud-core/src/main/java/com/google/cloud/Timestamp.java

```diff
@@ -170,7 +181,10 @@ public static Timestamp fromProto(com.google.protobuf.Timestamp proto) {
    * the timezone offset (always ends in "Z").
    */
   public static Timestamp parseTimestamp(String timestamp) {
-    Instant instant = Instant.parse(timestamp);
+    TemporalAccessor temporalAccessor = timestampFormatter.parse(timestamp);
+    LocalDateTime localDateTime = LocalDateTime.from(temporalAccessor);
+    ZonedDateTime zonedDateTime = ZonedDateTime.of(localDateTime, ZoneOffset.UTC);
```

Comment:
Should our parser have settings to automaticcaly be in `ZoneOffset.UTC`?

Reply:
Done.